src/item/filter.py

```diff
@@ -284,14 +284,14 @@ def should_keep(self, item: Item) -> FilterResult:
                         filter_min_affix_count = (
                             filter_data["minAffixCount"]
                             if "minAffixCount" in filter_data and filter_data["minAffixCount"] is not None
-                            else 0
+                            else 1
```

Comment:
I see the point to make it behave the same way as the minAffixCount in any_of blocks. Just note to self: Put a note in the patch notes so users know that this has changed!

Reply:
Should also mention the release notes that affixes are no longer removed from match. Previously something like
```
affixPool:
    - any_of:
      - fire_resistance
      - shadow_resistance
      - poison_resistance
      - lightning_resistance
      - cold_resistance
      minAffixCount: 1
    - any_of:
      - fire_resistance
      - shadow_resistance
      - poison_resistance
      - lightning_resistance
      - cold_resistance
      minAffixCount: 1
```

Would have not matched an item with just fire res, since you removed the affix from the pool on match. This is not done anymore, so doing it like above would match it. This is somewhat conceived since you now would just do

```
affixPool:
    - any_of:
      - fire_resistance
      - shadow_resistance
      - poison_resistance
      - lightning_resistance
      - cold_resistance
      minAffixCount: 2
```

and be done with it